src/modules/client/components/customers/infos/ProfileInfo.vue

```diff
@@ -887,20 +852,9 @@ export default {
         NotifyNegative('Erreur lors du téléchargement du devis.');
       }
     },
-    formatSubscriptionToGenerateQuote (subscription) {
-      const sub = {
-        serviceName: subscription.service.name,
-        unitTTCRate: subscription.unitTTCRate,
-        estimatedWeeklyVolume: subscription.estimatedWeeklyVolume,
-      };
-      if (subscription.sundays) sub.sundays = subscription.sundays;
-      if (subscription.evenings) sub.evenings = subscription.evenings;
-
-      return sub;
-    },
     async generateQuote () {
       try {
-        const subscriptions = this.subscriptions.map(this.formatSubscriptionToGenerateQuote);
+        const subscriptions = this.subscriptions.map(getSubscriptionQuoteTags);
         const payload = { subscriptions };
```

Comment:
tu pourrais simplifier : 
`const payload = { subscriptions: this.subscriptions.map(...) };`

Reply:
done